src/api/api.js

```diff
@@ -1 +1,5 @@
-export const BASE_URL = 'https://mate-api.herokuapp.com';
+export const baseUrl = 'https://mate-api.herokuapp.com';
+
+export const request = (url, option = {}) => fetch(baseUrl + url, option)
+  .then(response => response.json())
+  .then(data => data.data);
```

Comment:
```suggestion
  .then(result => result.data);
```
А то странно смотрится `data.data`